corehq/apps/dashboard/models.py

```diff
@@ -229,8 +229,6 @@ def _paginated_items(self, items_per_page, skip):
                 urlname = 'new_export_download_forms'
             elif isinstance(export, CaseExportSchema):
                 urlname = 'export_download_cases'
-            elif isinstance(export, FormExportSchema):
-                urlname = 'export_download_forms'
```

Comment:
Same as before - this is now unreferenced, so I can delete the view and merge its contents down to the subclass.